packages/selinux-policy/fs.cil

```diff
@@ -47,6 +47,7 @@
 (filecon "/.*/usr/bin/storewolf" file api_exec)
 (filecon "/.*/usr/bin/cfsignal" file api_exec)
 (filecon "/.*/usr/bin/thar-be-settings" file api_exec)
+(filecon "/.*/usr/bin/hachiko" file api_exec)
```

Comment:
@bcressey there are a few programs now that are `api_exec` just to let them write to the persistent partition. Do you think it will be worth having a less privileged label that grants this permission for programs like `cfsignal` or `hachiko`?